monitoring/micrometer-prometheus/src/test/java/io/quarkus/ts/micrometer/prometheus/HttpPathMetricsPatternIT.java

```diff
@@ -0,0 +1,95 @@
+package io.quarkus.ts.micrometer.prometheus;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class HttpPathMetricsPatternIT {
+
+    private static final int ASSERT_METRICS_TIMEOUT_MINUTES = 1;
+    private static final List<String> HTTP_SERVER_REQUESTS_METRICS_SUFFIX = Arrays.asList("count", "sum", "max");
+    private static final String HTTP_SERVER_REQUESTS_METRICS_FORMAT = "http_server_requests_seconds_%s{method=\"GET\",outcome=\"%s\",status=\"%d\",uri=\"%s\"}";
+    private static final String REDIRECT_ENDPOINT = "/test/redirect";
+    private static final String NOT_FOUND_ENDPOINT = "/test/not-found";
+    private static final String MOVED_ENDPOINT = "/test/moved/{id}";
+    private static final String EMPTY_ENDPOINT = "/test/";
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .withProperty("quarkus.management.enabled", "true");
+
+    @Test
+    public void testNotFoundEndpoint() {
+        whenCallNotFoundEndpoint();
+        thenMetricIsExposedInServiceEndpoint(404, "CLIENT_ERROR", NOT_FOUND_ENDPOINT);
+    }
+
+    @Test
+    public void testRedirectEndpoint() {
+        whenCallRedirectEndpoint();
+        thenMetricIsExposedInServiceEndpoint(302, "REDIRECTION", REDIRECT_ENDPOINT);
```

Comment:
```suggestion
        thenMetricIsExposedInServiceEndpoint(302, "REDIRECTION", "REDIRECTION");
```
fails because there is no template; please create additional dynamic redirect test that will test expected URI to be `REDIRECT_ENDPOINT`.